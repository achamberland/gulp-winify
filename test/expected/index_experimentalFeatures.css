/*Class */

.¢ {
  color: #0f0;
}

/* Tag with class */

¢.¢ {
  border: 1px solid #66f;
}

/* Class with tag */

.£ a {
  text-decoration: none;
}

/* Tag with child class */

£ .¢ {
  background: #ff6;
}

/* Multi-selector rules */

.¢ a,
.¢ p,
.¢ h1 {
  font-weight: 700;
}

/* :not pseudo-selectors with class */

.¢:not(.¤) {
  opacity: 1;
}

/* :not pseudo-selectors with another pseudo-selector */

£ ¢.¢:not(:hover) {
  color: #f00;
}

/* Pseudo-selector with a :not pseudo-selector & class */

£ ¢:hover:not(.¢) {
  color: #ff0;
}

/* To add and support: attr[class='...'] and all other alt equality signs */

#page-top {
  visibility: hidden;
}

¤ {
  background: 'hot pink';
}

£ ¥ ¢ {
  list-style: none;
}