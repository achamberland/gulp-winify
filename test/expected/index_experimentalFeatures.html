<!--<![CDATA[--><!doctype html>
<!--]]-->
<html>

<head>

  <!-- External styles -->
  <link rel="stylesheet" src="build/demo.css">

  <!-- External scripts -->
  <link rel="stylesheet" src="build/demo.css">


  <!-- Embedded styles -->
  <style>/* Tag styles */

¢ {
  border: 1px solid #000;
}

£ ¤ {
  display: inline-block;
}

/* Class styles */

.¢ {
  background: #000;
}

/* Id styles */

#top {
  float: right;
  padding: 20px;
  background: "hot pink";
}

/* Combo styles */

£ .£ {
  color: #f66;
}

¥ #article-top {
  color: #66f;
}

</style>

  <!-- Embedded scripts -->
  <script type="text/javascript">
    
    var d = document.registerElement('d', {
      prototype: Object.create(HTMLDivElement.prototype)
    });

  </script>

</head>

<body>

  <¢ class="container ¢">


  <¦>
    <£>
      <§>
        <¤ class="active £">
          <a href="#" id="Home">Home</a>
        </¤>
        <¤>
          <a href="#" id="About">About</a>
        </¤>
      </§>
    </£>
  </¦>

  <¥>
    <¢ id="page-top">Top of article</¢>

    <h1></h1>

    <¢ class="shady ¤ another-class ¥">
      Shaded text
    </¢>

    <¨ class="bold ¦" style="display:none"> </¨>



    <© class="class1 §"></©>
    <ª class="class11 ¨"></ª>
    <«>
    <¬>
    <®>
    <¯>
    <°>
    <±>
    <²>
    <³>
    <´>
    <µ>
    <¶>
    <·>
    <¸>
    <¹>
    <º>
    <»>
    <¼>
    <½>
    <©></©>
    <¾>
    <¿>
    <À>
    <Á>
    <Â>
    <Ã>
    <Ä>
    <Å>
    <Æ>
    <Ç>
    <È>
    <É>
    <Ê>
    <Ë>
    <Ì>
    <Í>
    <Î>
    <Ï>
    <Ð>
    <Ñ>
    <Ò>
    <Ó>
    <Ô>
    <Õ>
    <Ö>
    <×>
    <Ø>
    <Ù>
    <Ú>
    <Û>
    <Ü>
    <Ý>
    <×>
    <Þ>
    <ß>
    <à>
    <á>
    <â>
    <ã>
    <ä>
    <å>
    <æ>
    <ç>
    <è>

      <!-- Unclosed/misspelled tag -->
      <¢>

    </¢></è></ç></æ></å></ä></ã></â></á></à></ß></Þ></×></Ý></Ü></Û></Ú></Ù></Ø></×></Ö></Õ></Ô></Ó></Ò></Ñ></Ð></Ï></Î></Í></Ì></Ë></Ê></É></È></Ç></Æ></Å></Ä></Ã></Â></Á></À></¿></¾></½></¼></»></º></¹></¸></·></¶></µ></´></³></²></±></°></¯></®></¬></«></¥>

  </¢>

</body>


</html>